app/components/dynamic-content/dynamic-content-form/dynamic-content-form-field/dynamic-content-form-field-boolean/dynamic-content-form-field-boolean.html

```diff
@@ -0,0 +1,13 @@
+<dynamic-content-form-field-input-label
+    name="{{ name }}"
+    schema="schema">
+</dynamic-content-form-field-input-label>
+<input
+    name="{{ name }}"
+    ng-model="ngModel"
+    type="checkbox"
+    ng-required="schema.required">
+<dynamic-content-form-field-input-errors
```

Comment:
@kand Do we need the field errors here?

Boolean should be either true or false. Is there a case for a `required` error here?

Will there be a readonly case for this input?